server/aws-lsp-yaml-json/src/language-server/testUtils.ts

```diff
@@ -1,23 +1,58 @@
 import { TextDocument } from 'vscode-languageserver-textdocument'
+import { TextDocumentItem } from 'vscode-languageserver-types'
 
-const JSON_COMPLETION_CONTENT = `{
+const json_completion_content = `{
```

Comment:
nit: is there a reason for using different letters casing for constants in this file?

Reply:
I've just made name in lower case for all constants that are not exported from this file. If it looks ugly, I cat revert it

Reply:
It's up to you, I just think consistency is better